packages/nimble-components/src/select/tests/select.spec.ts

```diff
@@ -500,6 +500,35 @@ describe('Select', () => {
         await disconnect();
     });
 
+    it('option added directly to DOM under group synchronously registers with Select', async () => {
+        const { element, connect, disconnect } = await setupWithGroups();
+        await connect();
+        await waitForUpdatesAsync();
+        const newOption = new ListOption('foo', 'foo');
+        const registerOptionSpy = spyOn(
+            element,
+            'registerOption'
+        ).and.callThrough();
+        registerOptionSpy.calls.reset();
```

Comment:
do you need to reset a freshly made spy?

Reply:
Nope.